### PR TITLE
Documentation and redundant parameters in terraform.

### PR DESCRIPTION
**Pull request by benwarfield-usds**
Deleted or commented out all the places we set the database schema to the same value it is set to in the application configs, and the places we set spring.liquibase.enabled to its default value.

### CODE REVIEW

**Comment by benwarfield-usds** [comment]
I would beg forgiveness for the extensive whitespace-only changes, but they are 100% not my fault!

File: ops/demo/api.tf [hunk 1]
```diff
@@ -12,17 +12,19 @@ module "simple_report_api" {
...
-    DATAHUB_API_KEY                                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.datahub_api_key.id})"
-    SECRET_SLACK_NOTIFY_WEBHOOK_URL                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.slack_notify_webhook_url.id})"
-    TWILIO_ACCOUNT_SID                             = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.twilio_account_sid.id})"
-    TWILIO_AUTH_TOKEN                              = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.twilio_auth_token.id})"
-    SIMPLE_REPORT_SENDGRID_API_KEY                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.sendgrid_api_key.id})"
-    SMARTY_AUTH_ID                                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.smarty_auth_id.id})"
-    SMARTY_AUTH_TOKEN                              = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.smarty_auth_token.id})"
+    SPRING_PROFILES_ACTIVE                = "azure-demo"
```
Comment (jdorothy) [inline ops/demo/api.tf:15]: The change to the active profiles here were intentional for demo?

File: ops/prod/api.tf [hunk 2]
```diff
@@ -14,22 +14,24 @@ module "simple_report_api" {
...
-    ORG_EXTERNAL_ID                                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.org_external_id.id})"
-    ORG_CLIA_NUMBER                                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.org_clia_number.id})"
-    TWILIO_ACCOUNT_SID                             = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.twilio_account_sid.id})"
-    TWILIO_AUTH_TOKEN                              = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.twilio_auth_token.id})"
-    SIMPLE_REPORT_SENDGRID_API_KEY                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.sendgrid_api_key.id})"
-    SMARTY_AUTH_ID                                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.smarty_auth_id.id})"
-    SMARTY_AUTH_TOKEN                              = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.smarty_auth_token.id})"
+    SPRING_PROFILES_ACTIVE = "azure-prod"
+    # true by default: can be disabled quickly here
```
Comment (jdorothy) [inline ops/prod/api.tf:18]: Can we move this comment block to match the other envs?

**Review by jdorothy — CHANGES_REQUESTED** [review]
Looks good, had one question

File: ops/demo/api.tf (hunk 1 above)
Comment (benwarfield-usds) [reply ops/demo/api.tf:15]: Yes, the profiles other than the main profile for the environment should all be (and are all) set in Spring Boot configuration. This actually changes nothing, it just eliminates duplication.

File: ops/prod/api.tf (hunk 2 above)
Comment (benwarfield-usds) [reply ops/prod/api.tf:18]: D'oh.

Comment (jdorothy) [reply ops/prod/api.tf:18]: Ah, got it.

**Review by jdorothy — APPROVED** [review]

**Pull request merged**